cluster/examples/kubernetes/ceph/pool.yaml

```diff
@@ -54,6 +54,12 @@ spec:
     mirror:
       disabled: false
       interval: 60s
+  # quota in bytes and/or objects, default value is 0 (unlimited)
+  # see https://docs.ceph.com/en/latest/rados/operations/pools/#set-pool-quotas
+  quotas:
+    enabled: false
+    # maxBytes: 10737418240 # 10GiB
```

Comment:
Instead of a raw number, how about a string and allowing `10G` or similar to be specified? For example, in the OBCs there is a `maxSize` that allows for the size notation like that. See [this topic](https://rook.github.io/docs/rook/v1.5/ceph-object-bucket-claim.html), though I don't recall how it was implemented.

Reply:
I was thinking about that but it seemed like it could become an open can of worms. Let me go though the link.

Reply:
It would also mean that we would lose the ability to verify that the user is only using a non-zero integer in the CR. It would need to be type `string` which I imagine could lead to unforeseen issues. Maybe it would be best to let Ceph take the lead on that one.

Reply:
It looks like the objectBucket quotas are configured via RadosGW which does accept input in the form of K/M/G etc:

https://docs.ceph.com/en/latest/radosgw/admin/#quota-management

Given that the input verification/conversion is being handled via RadosGW I would prefer to leave this PR as is and require the same input as Ceph. Would this be ok?

Reply:
This can use a Kubernetes `resource.Quantity` the same way that `ResourceLists` does in Kubernetes https://github.com/kubernetes/kubernetes/blob/3a22c023b9f1a7344ad96343cd68bc7409aee4ba/pkg/apis/core/types.go#L4252

Reply:
I actually prefer knowing that the quantity the user supplies will be a valid non-zero integer. This will also eliminate any possible confusion between say GB and GiB. Additionally, I feel like it is not worth the additional effort (at least for me). If someone else really wants the feature in the future they can open another PR. Sound fair?

Reply:
Yes, that sounds fine to me.